easybuild/easyconfigs/t/tesseract/tesseract-5.3.0-GCCcore-11.3.0.eb

```diff
@@ -0,0 +1,63 @@
+easyblock = 'CMakeMake'
+
+name = 'tesseract'
+version = '5.3.0'
+
+homepage = 'https://github.com/tesseract-ocr/tesseract'
+description = """Tesseract is an optical character recognition engine"""
+
+toolchain = {'name': 'GCCcore', 'version': '11.3.0'}
+
+github_account = 'tesseract-ocr'
+source_urls = [GITHUB_SOURCE]
+sources = [
+    '%(version)s.tar.gz',
+    {
+        'source_urls': ['https://github.com/tesseract-ocr/tessdata_best/archive/'],
+        'download_filename': '4.1.0.tar.gz',
+        'filename': 'tessdata_best-4.1.0.tar.gz',
+    },
+]
+checksums = [
+    {'5.3.0.tar.gz': '7e70870f8341e5ea228af2836ce79a36eefa11b01b56177b4a8997f330c014b8'},
+    {'tessdata_best-4.1.0.tar.gz': 'bb05b738298ae73e7130e2913ed002b49d94cd1cea508e63be1928fe47770b32'},
+]
+
+builddependencies = [
+    ('CMake', '3.24.3'),
+    ('binutils', '2.38'),
+    ('pkgconf', '1.8.0')
+]
+
+dependencies = [
+    ('zlib', '1.2.12'),
+    ('libpng', '1.6.37'),
+    ('libjpeg-turbo', '2.1.3'),
+    ('LibTIFF', '4.3.0'),
+    ('Leptonica', '1.83.0'),
+    ('libarchive', '3.6.1'),
+    ('ICU', '71.1'),
+    ('fontconfig', '2.14.0'),
+    ('GLib', '2.72.1'),
+    ('cairo', '1.17.4'),
+    ('Pango', '1.50.7'),
+]
+
+separate_build_dir = True
+
+postinstallcmds = [
+    'rm %(builddir)s/tessdata_best-4.1.0/configs',
+    'rm -rf %(builddir)s/tessdata_best-4.1.0/tessconfigs',
+    'mv %(builddir)s/tessdata_best-4.1.0/* %(installdir)s/share/tessdata'
+]
+
+modextrapaths = {
+    'TESSDATA_PREFIX': 'tessdata',
+}
+
+sanity_check_paths = {
+    'files': ['bin/tesseract'],
+    'dirs': ['share/tessdata', 'include/tesseract']
+}
+
```

Comment:
Could you please include one or more simple sanity check commands for these modules, to verify that the executables can actually be run (e.g. `tesseract --version` or something)?

Reply:
Done! Actually found, that I needed to correct `TESSDATA_PREFIX`, so thank you! ;)